doc/Language/filename-extensions.rakudoc

```diff
@@ -0,0 +1,49 @@
+=begin pod :kind("Language") :subkind("Language") :category("beginning")
+
+=TITLE Filename extensions
+
+=SUBTITLE The extensions recommended for files with Raku content.
+
+=begin table
+File contents      | Extensions recommended or used at one time | Current recommendation
+===================================================================================
+B<Language files>  |                                            |
+Raku script        | .pl, .p6                                   | .raku
+Raku module        | .pm, .pm6                                  | .rakumod
+Raku documentation | .pm, pm6, pod, pod6                        | .rakudoc
```

Comment:
Did it ever use `.pm`?

Reply:
Not sure .pm was ever used for documentation, but Pod::To::HTML allowed for it. So on this basis pm/pm6 were once considered acceptable.